src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1,4 +1,20 @@
-// Add the required types and props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
-);
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+import { UserInfo } from '../UserInfo';
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  return (
+    <article className={classNames('TodoInfo', {
+      'TodoInfo--completed': todo.completed === true,
```

Comment:
completed - it's a boolean
```suggestion
      'TodoInfo--completed': todo.completed,
```